tests/testsuite/kubevirtresource.go

```diff
@@ -142,17 +142,16 @@ func AdjustKubeVirtResource() {
 	Expect(err).NotTo(HaveOccurred())
 	if checks.HasFeature(virtconfig.CPUManager) && len(nodes.Items) > 1 {
 		// CPUManager is not enabled in the control-plane node
-		waitForSchedulableNodeWithCPUManager()
+		waitForSchedulableNodesWithCPUManager(len(nodes.Items) - 1)
```

Comment:
There might be nodes that don't have a cpu manager. The alternative would be to check if every handler has the latest config, WDYT?

Reply:
Every non-control-plane should have cpumanager enabled by us, but I was indeed assuming only 1 control-plane node, fixed!